### PR TITLE
fastem-improvements-acq-calib

### PR DESCRIPTION
**Pull request by Sabrina85**
See descriptions of commits.

### CODE REVIEW

File: src/odemis/acq/test/fastem_test.py [hunk 1]
```diff
@@ -298,8 +298,8 @@ class TestFastEMAcquisition(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        if TEST_NOHW:
-            test.start_backend(FASTEM_CONFIG_ASM)
+        # if TEST_NOHW:
+        #     test.start_backend(FASTEM_CONFIG_ASM)
```
Comment (tepals) [inline src/odemis/acq/test/fastem_test.py:302]: I think this is from debugging?

Comment (Sabrina85) [reply src/odemis/acq/test/fastem_test.py:302]: yes

**Review by tepals — APPROVED** [review]

File: src/odemis/acq/fastem.py [hunk 2]
```diff
@@ -515,12 +511,17 @@ def pre_calibrate(self):
...
+            else:
+                configure_hw.configure_asm(self._multibeam, self._descanner, self._detector,
+                                           self._detector.dataflow, asm_config)
+            # put system back into state ready for next task and set calibrated settings
             configure_hw.configure_asm(self._multibeam,
                                        self._descanner,
                                        self._detector,
                                        self._detector.data,
-                                       asm_config_orig,
+                                       asm_config,
```
Comment (pieleric) [inline src/odemis/acq/fastem.py:524]: Why do you do the same thing twice in a row?
Also `self._detector.dataflow` probably doesn't exist, it's `self._detector.data`.

**Review by pieleric — CHANGES_REQUESTED** [review]

File: src/odemis/acq/fastem.py (hunk 2 above)
Comment (Sabrina85) [reply src/odemis/acq/fastem.py:524]: This indeed is incorrect.

**Review by pieleric — APPROVED** [review]

**Pull request merged**